### PR TITLE
3.0.0 Update

### PR DESCRIPTION
**Pull request by BootleggerFinn**
Removed Dr. Jekyll, added Close Encounters, added Patch Notes

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -203,7 +201,7 @@
     0,
     0
   ],
-  "SaveName": "Arkham SCE - 3.0",
+  "SaveName": "Arkham SCED - 3.0.0",
```
Comment (Chr1Z93) [inline config.json:204]: Is this name change intended? Savegames have usually been called "SCE"
Arkham SCE 2 4 0

File: objects/ArkhamSCED300-1262023-Page1.f873a8.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    "y": 0,
+    "z": 0
+  },
+  "Autoraise": true,
+  "ColorDiffuse": {
+    "b": 1,
+    "g": 1,
+    "r": 1
+  },
+  "Description": "- Revamped the entire table!\n- Revamped (almost) all official scenarios to use dynamic location marking!!!\n- Added an introductory tour by Joe Sargent and the Investigators of Arkham!!!!!\n- Revamped the deck construction! Cards and investigators can now be laid out by cycle as well as by class. (For now, Gloria Goldberg is with the Investigator Packs.)\n",
```
Comment (Chr1Z93) [inline objects/ArkhamSCED300-1262023-Page1.f873a8.json:13]: I share the excitement, but would cut down on the exclamation mark usage ;)

File: objects/ArkhamSCED300-1262023-Page1.f873a8.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  "Hands": false,
+  "HideWhenFaceDown": false,
+  "IgnoreFoW": false,
+  "LayoutGroupSortIndex": 0,
+  "Locked": false,
+  "LuaScript": "",
+  "LuaScriptState": "",
+  "MeasureMovement": false,
+  "Name": "Notecard",
+  "Nickname": "Arkham SCED 3.0.0 - 1/26/2023 - Page 1",
```
Comment (Chr1Z93) [inline objects/ArkhamSCED300-1262023-Page1.f873a8.json:28]: See above comment regarding name

**Review by Chr1Z93 — CHANGES_REQUESTED** [review]

File: config.json (hunk 1 above)
Comment (BootleggerFinn) [reply config.json:204]: Oh, good catch! Thank you

File: objects/ArkhamSCED300-1262023-Page1.f873a8.json (hunk 2 above)
Comment (BootleggerFinn) [reply objects/ArkhamSCED300-1262023-Page1.f873a8.json:13]: Fiiiiiine

**Review by Chr1Z93 — APPROVED** [review]

**Pull request merged**